generate_icons.sh

```diff
@@ -3,11 +3,12 @@
 icon_name="icon.png"
 output_path="icon.iconset"
 cd "$(dirname "${0}")" || exit
-rm -rf "output_path/*" || true
+rm -rf "$(output_path)/*" || true
```

Comment:
I think this is meant to be a variable ref?

```suggestion
rm -rf "${output_path}/*" || true
```


`shellcheck` also gives this warning (even before the suggested change)

```
rm -rf "${output_path}/*" || true
       ^----------------^ SC2115 (warning): Use "${var:?}" to ensure this never expands to /* .
```

Reply:
I can change this to just `${output_path}`, i don't think it needs the `/*`